core/tests/protractor_utils/forms.js

```diff
@@ -523,7 +529,9 @@ var RichTextChecker = async function(arrayOfElems, arrayOfTexts, fullText) {
       var elem = await arrayOfElems.get(arrayPointer);
       expect(await elem.getTagName()).
         toBe('oppia-noninteractive-' + componentName.toLowerCase());
-      expect(await elem.getText()).toBe(arrayOfTexts[arrayPointer]);
+      if (componentName !== 'Math') {
```

Comment:
This needs, at the very least, a comment explaining what's going on.

But still it's not so good. Can you find a positive way to check something useful in the "Math" case, rather than skipping it entirely?

Reply:
done